docs/Using-Pantheon/Debugging.md

```diff
@@ -52,26 +55,72 @@ block of the `prometheus.yml` file:
         using [`--host-whitelist`](../Reference/Pantheon-CLI-Syntax.md#host-whitelist) when starting Pantheon. 
 
 
-3. Start Pantheon with the [`--metrics-enabled` option](../Reference/Pantheon-CLI-Syntax.md#metrics-enabled). To start
+1. Start Pantheon with the [`--metrics-enabled` option](../Reference/Pantheon-CLI-Syntax.md#metrics-enabled). To start
  a single node for testing with metrics enabled:
 
     ```bash tab="Example"
     pantheon --network=dev --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73
     --rpc-http-cors-origins="all" --rpc-http-enabled --metrics-enabled
     ```
 
-4. In another terminal, run Prometheus specifying the `prometheus.yml` file: 
+1. In another terminal, run Prometheus specifying the `prometheus.yml` file: 
 
     ```bash tab="Example"
     prometheus --config.file=config.yml 
     ```
 
-5. Open a web browser to `http://localhost:9090` to view the Prometheus graphical interface.
+1. Open a web browser to `http://localhost:9090` to view the Prometheus graphical interface.
 
-6. Choose **Graph** from the menu bar and click the **Console** tab below.
+1. Choose **Graph** from the menu bar and click the **Console** tab below.
 
-7. From the **Insert metric at cursor** drop-down, select a metric such as `pantheon_blockchain_difficulty_total` or
+1. From the **Insert metric at cursor** drop-down, select a metric such as `pantheon_blockchain_difficulty_total` or
 `pantheon_blockchain_height` and click **Execute**. The values are displayed below.
 
     Click the **Graph** tab to view the data as a time-based graph. The query string is displayed below the graph. 
     For example: `{pantheon_blockchain_height{instance="localhost:9545",job="prometheus"}`
+
+### Running Prometheus with Pantheon in Push Mode 
+
+By default, Prometheus polls Pantheon for metrics but sometimes metrics are hard to poll (for example, 
+when running inside Docker containers with varying IP addresses). The `push` mode supports Pantheon 
+pushing metrics to a [Prometheus Pushgateway](https://github.com/prometheus/pushgateway).   
+
+To configure Prometheus and run with Pantheon pushing to a push gateway: 
+
+1. Configure Prometheus for Pantheon to push to the push gateway. For example, add the following yaml fragment to the `scrape_configs`
+   block of the `prometheus.yml` file:
+    
+       ```yml tab="Example"
```

Comment:
this sample doesn't not work because of YAML syntax issues. Use:
```
scrape_configs:
  - job_name: 'pantheon-dev'
    metrics_path: /metrics
    static_configs:
      - targets:
        - localhost:9091
```